### PR TITLE
Update installing.md - clarify instructions for Linux and ARM Macs

### PR DESCRIPTION
**Pull request by echuber2**
The network tweak for Linux, and the fix for ARM Macs, were missing from the extended workspace invocations.

### CODE REVIEW

**Comment by trombonekenny** [comment]
I don't need `--add-host` for docker on my Ubuntu dev box.

**Comment by echuber2** [comment]
Hm, I thought it was necessary on some distros. Maybe different versions of Docker mitigated the issue.

**Comment by SethPoulsen** [comment]
Having just tested the Linux commands myself, happy to approve, but we should probably wait for an approval from Nathan or Matt, who are the mac users.

**Review by SethPoulsen — APPROVED** [review]

File: docs/installing.md
Comment (echuber2) [reply docs/installing.md:132]: We could make a separate paragraph about it, but I'm also confused about how the bridge works differently on Mac. Maybe Matt and Nathan can confirm if the `--add-host` is needed at all on macOS.

Comment (mwest1066) [reply docs/installing.md:132]: It's not needed on MacOS.

Comment (echuber2) [reply docs/installing.md]: OK, I left it exclusive to the Linux/Windows section but I did move it to a separate paragraph.

**Comment by echuber2** [comment]
@SethPoulsen In light of Jonatan's comment, I broke out the note to a separate paragraph. Does it still look okay?

File: docs/installing.md
Comment (echuber2) [reply docs/installing.md]: I didn't notice that before, but in the latest commit, I have "address".

File: docs/installing.md
Comment (nwalters512) [inline docs/installing.md:63]:
```suggestion
### Running on Apple Silicon and other ARM64 hardware
```

File: docs/installing.md
Comment (nwalters512) [inline docs/installing.md:122]:
```suggestion
##### Linux or Windows WSL 2
```

File: docs/installing.md
Comment (nwalters512) [inline docs/installing.md:104]: Could we get away with combining these into one command for all platforms? Adding `--platform linux/amd64` unconditionally should be fine, and in my testing, `--add-host=host.docker.internal:172.17.0.1` didn't break anything on macOS.

File: docs/installing.md
Comment (echuber2) [reply docs/installing.md:63]: I can make this change, but I tried to refer to "M1" in the body text since not everybody knows that Apple calls their chip line "Apple Silicon". Apple themselves don't capitalize the word "silicon" and they didn't try to trademark it, from what I can tell, so it's sort of vague whether they actually mean this to be the product line's name, or they just expect people to understand that silicon means "chips".

File: docs/installing.md
Comment (echuber2) [reply docs/installing.md]: OK, I've tried combining the platform invocations into a single use case (latest commits).

File: docs/installing.md
Comment (nwalters512) [reply docs/installing.md]: You can lowercase silicon, or refer to "Apple M-series processors", but I don't want to hardcode references to specific versions (M1 or otherwise).

Comment (echuber2) [reply docs/installing.md]: I made it lowercase and changed "M1, M2, etc." to "M-series chips" in the body text. Coincidentally Apple also insists on referring to the Intel processors as "processors" and Apple silicon as "chips," not "processors." I don't mind any of this but it's amusing to note.

**Review by nwalters512 — APPROVED** [review]
I tested the command verbatim on macOS and it works great. Assuming it works on Linux and WSL 2 (I don't have access to a machine to test it), let's merge this!

**Comment by SethPoulsen** [comment]
I just double checked the command on Linux, good to go!

**Pull request merged**
